global-library-examples/reload-cps-library-on-scm-change/Jenkinsfile

```diff
@@ -0,0 +1,8 @@
+#!groovy
+
+node('master') {
+  stage "checkout"
+  checkout scm
+  stage "push to jenkins"
+  sh "git push jenkins@localhost:workflowLibs.git '+refs/remotes/origin/*:refs/heads/*'"
```

Comment:
I think this needs a lot of documentation — this won't work at all out of the box.

For this to work, you would need to enable the SSH server inside Jenkins, and have set the SSH server to run on port 22 (which I guess would require some pretty special setup on that machine), and have security disabled.

If security is enabled, then you'd need to add a `jenkins` user to Jenkins, create a key pair for them in `$JENKINS_HOME/.ssh` on the master, add the corresponding public key to Jenkins, and have `git` installed on the master.